qt/aqt/models.py

```diff
@@ -49,20 +49,26 @@ def setupModels(self) -> None:
         self.model = None
         f = self.form
         box = f.buttonBox
-        t = QDialogButtonBox.ActionRole
-        b = box.addButton(_("Add"), t)
-        qconnect(b.clicked, self.onAdd)
-        b = box.addButton(_("Rename"), t)
-        qconnect(b.clicked, self.onRename)
-        b = box.addButton(_("Delete"), t)
-        qconnect(b.clicked, self.onDelete)
+
+        default_buttons = [
+            ("Add", self.onAdd),
+            ("Rename", self.onRename),
+            ("Delete", self.onDelete),
+            ("Options...", self.onAdvanced),
+        ]
+
         if self.fromMain:
-            b = box.addButton(_("Fields..."), t)
-            qconnect(b.clicked, self.onFields)
-            b = box.addButton(_("Cards..."), t)
-            qconnect(b.clicked, self.onCards)
-        b = box.addButton(_("Options..."), t)
-        qconnect(b.clicked, self.onAdvanced)
+            from_main_buttons = [
+                ("Fields...", self.onFields),
+                ("Cards...", self.onCards),
+            ]
+
+            default_buttons[-1:-1] = from_main_buttons
```

Comment:
This is difficult to read; I'd prefer you just append or extend() these, and then append Options at the end